solidity/test/TokenHolder.js

```diff
@@ -1,52 +1,179 @@
 const { accounts, contract } = require('@openzeppelin/test-environment');
 const { expect } = require('../../chai-local');
-const { expectRevert, constants, BN } = require('@openzeppelin/test-helpers');
+const { expectRevert, constants, BN, balance } = require('@openzeppelin/test-helpers');
 
 const { ZERO_ADDRESS } = constants;
 
+const { NATIVE_TOKEN_ADDRESS } = require('./helpers/Constants');
+
 const TokenHolder = contract.fromArtifact('TokenHolder');
 const TestStandardToken = contract.fromArtifact('TestStandardToken');
 
 describe('TokenHolder', () => {
+    const getBalance = async (tokenAddress, account) => {
+        if (tokenAddress === NATIVE_TOKEN_ADDRESS) {
+            return balance.current(account);
```

Comment:
I have no idea what `balance` is, but I would imagine that we require an `await` in order to resolve the promise object that it returns.
I understand that omitting an `await` on `return` statement is technically fine, but just in case we change this in the future to an assignment into some temporary variable (for example), and forget that we need an `await` here, it is better that we stick to placing `await` before every asynchronous function call.

Reply:
Actually no `await`ing on returns from an `async` function is the preferred way and it's also more performarnt